vpdq/python/vpdq.pyx

```diff
@@ -125,12 +98,12 @@ def hamming_distance(hash1: "Hash256", hash2: "Hash256") -> int:
 
 def computeHash(
     input_video_filename: t.Union[str, Path],
-    ffmpeg_path: str = "ffmpeg",
+    ffmpeg_path: t.Union[str, None] = None,
```

Comment:
Question for you - I had a devil of a time getting any python tooling to run on the .pyx file. Did you get typechecking to run properly, or just keeping them updated by eye?

Reply:
I've just been keeping them updated by eye. Some of them could be wrong, but I doubt it.